python/README.md

```diff
@@ -1,73 +1,181 @@
 # Python Arrow Flight Client Application Example
 
-## Getting Started
-1. Install [Python 3](https://www.python.org/downloads/)
-2. Download and install the [dremio-flight-endpoint whl file](https://github.com/dremio-hub/arrow-flight-client-examples/releases)
-    - `python -m pip install <PATH TO WHEEL>` 
-3. Copy the contents of arrow-flight-client-examples/python/example.py into your own python file. 
-4. Run your python file with a local instance of Dremio:
-    - `python3 example.py --username <USER> --password <password> -query 'SELECT 1'`
+## Quickstart
 
+1. Install [Python 3](https://www.python.org/downloads/)
+1. Download and install the [dremio-flight-endpoint whl file](https://github.com/dremio-hub/arrow-flight-client-examples/releases)
+    - `python -m pip install <PATH TO WHEEL>`
+1. Create a file named `config.yaml` in the `arrow-flight-client-examples/python` directory. Copy the contents of `arrow-flight-client-examples/python/config_template.yaml` into `config.yaml`.
+1. Uncomment the options in `config.yaml` as needed, appending the argument after the key. ie. `username: my_username`. You can either delete the options that are not being used or leave them commented.
+    - e.g. If you are connecting to a local instance of Dremio, your config file would look like:
+        ```
+        username: my_username
+        password: my_password
+        query: SELECT 1
+        ```
+1. Run the Python Arrow Flight Client by navigating to the `arrow-flight-client-examples/python` directory and running the command `python3 example.py`.
```

Comment:
Probably need a step before this to copy the example.py file from the repo.

Reply:
I don't think there is a need to copy the file from the repo. It adds an extra step and provides no real benefit given that you can just run the file directly. In addition, users who want to make changes to the example client would know to copy the file into their own repo.

Reply:
What do you mean by run the file directly?

Reply:
Run example.py directly.

Reply:
How do they know what the content of example.py is?

Reply:
Nvm. I see now that we're assuming they clone the repo? If we're going that route, then we should add a step for cloning the repo as well (which I think adds more work for the user after they've already installed the python package).

Reply:
Ah I was working under the assumption that the repo was already cloned. In that case it makes sense to revert back to the copying of example.py.

Reply:
@ravjotbrar can you take another look and see if this new readme is better?